silver/admin.py

```diff
@@ -468,6 +474,21 @@ def __init__(self, *args, **kwargs):
 
         super(BillingDocumentForm, self).__init__(*args, **kwargs)
 
+    def clean(self, *args, **kwargs):
+        cleaned_data = super(BillingDocumentForm, self).clean(*args, **kwargs)
+
+        customer = cleaned_data['customer']
+        currency = cleaned_data['currency']
+
+        cleaned_data['transaction_currency'] = (
```

Comment:
Maybe add a validation for `transaction_currency` to be a value from `currencies`